File: src/main/java/baboon/industry/block/storage/entity/TileEntityBatboxLV.java
package baboon.industry.block.storage.entity;

import baboon.industry.IndustryConfig;

public class TileEntityBatboxLV extends TileEntityBatboxBase {

    public TileEntityBatboxLV() {
        setCapacity(IndustryConfig.cfg.getInt("Energy Values.lvBatteryStorage") * 3);
        setTransfer(IndustryConfig.cfg.getInt("Energy Values.lvIO"));
        setMaxReceive(IndustryConfig.cfg.getInt("Energy Values.lvIO"));
    }

    @Override
    public String getInvName() {
        return "IndustryBatboxLV";
    }
}

File: src/main/kotlin/turniplabs/industry/blocks/entities/batbox/TileEntityBatboxLV.kt
package turniplabs.industry.blocks.entities.batbox

import sunsetsatellite.sunsetutils.util.Connection
import sunsetsatellite.sunsetutils.util.Direction
import turniplabs.industry.IndustryConfig

class TileEntityBatboxLV : TileEntityBatboxBase() {

    init {
        this.setCapacity(8192)
        this.setTransfer(IndustryConfig.cfg.getInt("Energy Values.lowVoltage"))
        this.setMaxReceive(IndustryConfig.cfg.getInt("Energy Values.lowVoltage"))
    }

    override fun getInvName(): String {
        return "BatboxLV"
    }
}